src/OrchardCore.Modules/OrchardCore.Security/Middlewares/StrictTransportSecurityMiddleware.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using OrchardCore.Security.Options;
+
+namespace OrchardCore.Security.Middlewares
+{
+    public class StrictTransportSecurityMiddleware
+    {
+        private readonly StrictTransportSecurityOptions _options;
+        private readonly RequestDelegate _next;
+
+        public StrictTransportSecurityMiddleware(IOptions<StrictTransportSecurityOptions> options, RequestDelegate next)
+        {
+            _options = options.Value;
+            _next = next;
+        }
+
+        public Task Invoke(HttpContext context)
+        {
+            var value = "max-age=" + _options.MaxAge.TotalSeconds;
```

Comment:
This should be cached in a field, it never changes unless the tenant is restarted